app/src/components/template/Dashboard.js

```diff
@@ -93,6 +116,31 @@ class Dashboard extends Component {
                     <main className={classes.content}>
                         <div className={classes.appBarSpacer} />
                         <Container className={classes.container}>
+                            <InputLabel
+                                className={classes.selectInstrumentLabel}
+                                id={'selectInstrumentLabel'}
+                            >
+                                Select Instrument
+                            </InputLabel>
+                            <Select
+                                value={selectedInstrument}
+                                onChange={this.handleInstrumentChange}
+                                className={classes.selectInstrumentInput}
+                            >
+                                {
+                                    instruments.map(value => {
+                                        return (
+                                            <MenuItem
+                                                key={`menuitem-${value}`}
+                                                value={value}
+                                            >
+                                                {value}
+                                            </MenuItem>
+
+                                        );
+                                    })
+                                }
+                            </Select>
```

Comment:
Should this instrument selector only be proper to the OrderBook snapshot component in our app.
i.e. only visible when selecting on the side bar:

![image](https://user-images.githubusercontent.com/29107916/69471305-ea319080-0d6b-11ea-9ff8-47309c7c44d6.png)

Because right now it's visible at the root

Reply:
hmm, i Put it app wide. outside of the container used currently for displaying our components based on routing. So currently you will see it no matter what page you are on. How would we prefer it?

Reply:
think it makes more sense for it to be only visible once we click on orderbook tab

Reply:
kk 👍